common/character_interactions/00_blackmail_interactions.txt

```diff
@@ -12,7 +12,8 @@
 	is_shown = {
```

Comment:
For some reason, elves still get Infirm very often.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/114045265-f3f06100-9898-11eb-8115-127c9b2794f7.png)

</details>

Reply:
Nah.
 I think I just have bad luck. Humans get the Infirm at the same rate.

Reply:
Though, nope, just after 10 years, every elven ruler is Infirm.

Reply:
Fixed in 695c853e5be54af1455f21fc6599890243ac2a2f.